src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,61 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.EAGER)
+    private Movie movie;
+    @ManyToOne(fetch = FetchType.EAGER)
+    private CinemaHall cinemaHall;
+    @Column(name = "show_time")
+    private LocalDateTime showTime;
+
+    public MovieSession() {
+
+    }
```

Comment:
It's unnecessary to add a default constructor if you don't have other constructors

Reply:
remove it, since you do not have any other constructors - it is not needed